pkgs/by-name/ae/aerospace/package.nix

```
@@ -0,0 +1,32 @@
+{ stdenv, fetchzip, lib, callPackage }:
+
+let
+  pname = "aerospace";
+  # Version must be a literal string as package has a capital letter in it
+  # whilst nixpkgs forbids capitals in version names
+  ghReleaseVersion = "0.9.0-Beta";
+  version = lib.strings.toLower ghReleaseVersion;
+  meta = with lib; {
+    license = licenses.mit;
+    mainProgram = "aerospace";
+    homepage = "https://github.com/nikitabobko/AeroSpace";
+    description = "an i3-like tiling window manager for macOS";
+    platforms = [ "aarch64-darwin" ];
+    maintainers = with maintainers; [ t-monaghan ];
```

```suggestion
    maintainers = with maintainers; [ t-monaghan ];
    sourceProvenance = lib.sourceTypes.binaryNativeCode;
```

when using prebuilt binaries you must set the metadata that says as much

<https://nixos.org/manual/nixpkgs/unstable/#sec-meta-sourceProvenance>